packages/parity-electron/src/types.d.ts

```diff
@@ -3,14 +3,8 @@
 //
 // SPDX-License-Identifier: BSD-3-Clause
 
-export type CliObject = {
-  rawArgs?: string[];
-  [key: string]: string | boolean | string[];
-};
-
 export type LoggerFunction = (namespace: string) => (log: string) => void;
 
 export type Options = {
-  cli: CliObject;
```

Comment:
I think this Options type is not used anywhere, is it? Putting the type in index.ts's function argument is enough.